.github/workflows/playwright.yml

```diff
@@ -7,7 +7,7 @@ on:
       - completed
 
   pull_request:
-    branches: [ dev ]
```

Comment:
@Jamesunekwuojo 

How are your lint settings? 
This also seems like an unnecessary change.